package/MDAnalysis/lib/formats/libmdaxdr.pyx

```diff
@@ -19,6 +19,32 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
+"""Low-level Gromacs XDR trajectory reading — :mod:`MDAnalysis.lib.formats.libmdaxdr`
+---------------------------------------------------------------------------------
+
+libmdaxdr contains the classes :mod:`XTCFile` and :mod:`TRRFile`. Both can be
+used to read and write frames from to XTC and TRR files. These classes are used
+internally by MDAnalysis in :mod:`MDAnalysis.coordinates.XTC` and
+:mod:`MDAnalysis.coordinates.TRR`. They behave similar to normal file objects.
+For example to calculate mean coordinates.
+
+.. code::python
+  with XTFile("trajectory.xtc") as xtc:
+      n_atoms = xtc.n_atoms
+      mean = np.zeros((n_atoms, 3))
+      # iterate over trajectory
+      for frame in xtc:
+          mean += frame.x
+
+The XTFile class can be usefull as a compressed storage format.
```

Comment:
XTFile

Actually, mark it up: 
```reST
The :class:`XTCFile` class can be ...
```
 (assuming it lives in *this* module). It's really nice when you can directly click on the class to find the docs.